js/app.js

```diff
@@ -26,7 +26,7 @@ $(document).ready(function() {
 	});
 	$(document).keydown(function (event) {
 		if (event.which == 88) {
-			$('ryu.still').hide();
```

Comment:
@yotterh The typo is the main problem.  I think you can figure it out from here.